src/modules/rlm_smtp/rlm_smtp.c

```diff
@@ -0,0 +1,897 @@
+/*
+ *   This program is is free software; you can redistribute it and/or modify
+ *   it under the terms of the GNU General Public License as published by
+ *   the Free Software Foundation; either version 2 of the License, or (at
+ *   your option) any later version.
+ *
+ *   This program is distributed in the hope that it will be useful,
+ *   but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *   GNU General Public License for more details.
+ *
+ *   You should have received a copy of the GNU General Public License
+ *   along with this program; if not, write to the Free Software
+ *   Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301, USA
+ */
+
+/**
+ * $Id$
+ * @file rlm_smtp.c
+ * @brief smtp server authentication.
+ *
+ * @copyright 2020 The FreeRADIUS server project
+ * @copyright 2020 Network RADIUS SARL <legal@networkradius.com>
+ */
+RCSID("$Id$")
+
+#include <freeradius-devel/server/base.h>
+#include <freeradius-devel/server/module.h>
+#include <freeradius-devel/curl/base.h>
+
+static fr_dict_t 	const 	*dict_radius; /*dictionary for radius protocol*/
+fr_dict_t 		const 	*dict_freeradius;
+
+#define MAX_ATTRMAP	128
+
+extern fr_dict_autoload_t rlm_smtp_dict[];
+fr_dict_autoload_t rlm_smtp_dict[] = {
+	{ .out = &dict_radius, .proto = "radius" },
+	{ .out = &dict_freeradius, .proto = "freeradius"},
+	{ NULL }
+};
+
+static fr_dict_attr_t 	const 	*attr_auth_type;
```

Comment:
`const` is left associative, and should be part of the type on the LHS. i.e. `static fr_dict_attr_t const<tabs>*<variable name>`